lib/semian/protected_resource.rb

```diff
@@ -2,35 +2,59 @@ module Semian
   class ProtectedResource
     extend Forwardable
 
-    def_delegators :@resource, :destroy, :count, :semid, :tickets, :name
+    def_delegators :@bulkhead, :destroy, :count, :semid, :tickets, :name
```

Comment:
I agree, but this will break backwards compatibility—I'm afraid we'll have to alias and deprecate this.